src/models/user/postgres.js

```diff
@@ -86,11 +88,31 @@ const UserRepo = (postgres) => {
     }
   };
 
+  //Set the 
+  const setHoursSQL = `
+    UPDATE users
+    SET hours_start=$1, hours_end=$2
+    WHERE id=$3;`;
```

Comment:
Add `RETURNING *` to the end of the SQL, so that way it will return the updated user model directly from the request

Reply:
<img width="1173" alt="Screen Shot 2020-01-20 at 15 57 23" src="https://user-images.githubusercontent.com/11587143/72764867-a4901900-3b9e-11ea-8b38-299340ef76e4.png">